src/init.cpp

```diff
@@ -1562,6 +1573,7 @@ bool AppInitMain(const std::any& context, NodeContext& node, interfaces::BlockAn
         filter_index_cache = max_cache / n_indexes;
         nTotalCache -= filter_index_cache * n_indexes;
     }
+    int64_t coin_stats_cache_size = 0;
```

Comment:
13d2fc6 looks like this can be const (I'm not sure there is a reason to declare this variable instead of doing `std::make_unique<CoinStatsIndex>(/* coin_stats_cache_size */ 0, false, fReindex);` at line 1829 below)

Reply:
Done (inlined)